.gitignore

```diff
@@ -8,10 +8,12 @@ nmme/
 copernicus/
 chirps/
 analyses/bangladesh/data/*
+analyses/bangladesh/results/*/archive/*
+analyses/bangladesh/trigger_development/processed_data/*
```

Comment:
You mean the csv file? I think that could make sense although I'm worried it would be a bit confusing with the Glofas data and the skill results in different places. Maybe it'll make more sense to keep it in the ```processed``` folder once (in theory) this notebook becomes a reusable script in the ```src``` folder?

Reply:
Yes exactly, so I think the idea is that the `raw` and `processed` directories are intended for inputs and outputs of the main codebase. For one-off analyses, then the inputs and outputs can all go in `exploration`, and if the functions get integrated into the main codebase then the data can be moved. So I agree that the skills output should eventually end up in `processed`, but only once we have a proper pipeline in place for it. Let me know if that makes sense!